build.sbt

```diff
@@ -28,9 +28,9 @@ scalacOptions ++= Seq(
 parallelExecution in Test := false
 
 libraryDependencies ++= Seq(
-  "com.datastax.cassandra"  % "cassandra-driver-core"             % "2.1.5",
-  "com.typesafe.akka"      %% "akka-persistence-experimental"     % "2.3.11",
-  "com.typesafe.akka"      %% "akka-persistence-tck-experimental" % "2.3.11"  % "test",
-  "org.scalatest"          %% "scalatest"                         % "2.1.4"   % "test",
-  "org.cassandraunit"       % "cassandra-unit"                    % "2.0.2.2" % "test"
+  "com.datastax.cassandra"  % "cassandra-driver-core"             % "2.1.7",
+  "com.typesafe.akka"      %% "akka-persistence-experimental"     % "2.4-M2",
+  "com.typesafe.akka"      %% "akka-persistence-experimental-tck" % "2.4-M2"  % "test",
```

Comment:
sorry about that rename, it's accidental; M3 will remove the `experimental` part from persistence alltogether, so we're not rushing to fix the reordering here just yet.